### ISSUE TITLE
Problem with GI/Restart 12.2 on OL/RHEL7 with Kernel 3.10.0

### ISSUE THREAD

**Issue by Rendanic** [oravirt/ansible-oracle#56]
I am working on a solution for Note **2284463.1** who explains a problem with Kernel 3.10.0 on OL/RHEL7. The Kernel is the RHCK under OL and the normal kernel for RHEL.
A patch is requirred after the unzip of the installation media. Otherwise the configuration will fail with 
> CLSRSC-400: A system reboot is required to continue installing.

The patch from 2284463.1 is part of the last Release Update from January 2018. That's why I am working on a solution to install the RU during oraswgi-install for 12.2 for OL/RHEL 6 and 7.

This will change a lot in `roles/oraswgi-install/tasks/12.2.0.1.yml` but it is mandatory for a working installation of GI/Restart 12.2 on RHEL7.

**Comment by Rendanic** [comment]
I completed the development for this problem.
I created the changes on top of #49, because the pull request is mandatory for GI/Restart installation in 12.2. The pull request for this issue could be created after #49 has been approved.

**Comment by oravirt** [comment]
PR?

**Comment by Rendanic** [comment]
The changes are in 7e186410dcb0fe3e3eeca22bdc32c34f883f69a7 and part of #49. Interesting, because the  PR was created before the commit. Maybe I made a mistake during the request for the PR, because I tried to create another PR on top of the existing and github put them together.

We can close this bug, because the needed changes are commited to master.
